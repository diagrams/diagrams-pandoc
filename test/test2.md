Here is a square:

``` {.diagrams echo=Above}
example = square 1
```

