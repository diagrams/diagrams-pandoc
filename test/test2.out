[ Para
    [ Str "Here"
    , Space
    , Str "is"
    , Space
    , Str "a"
    , Space
    , Str "square:"
    ]
, CodeBlock
    ( "" , [ "haskell" ] , [ ( "echo" , "Above" ) ] )
    "example = square 1"
, Plain
    [ Image
        ( "" , [] , [] ) [] ( "images/3895b48f2a2f113a.png" , "" )
    ]
]